python/GafferUI/Slider.py

```diff
@@ -206,6 +214,64 @@ def getPositionIncrement( self ) :
 
 		return self.__positionIncrement
 
+	## Sets whether mouse interactions with the slider should snap to
+	# specific position increments. This does not apply to programmatic
+	# changes via setPosition(s) or keyboard increments.
+	def setSnaps( self, snaps ) :
```

Comment:
`setSnaps()` seems like an awkward phrasing, but I think we do seem to generally be avoiding the verbosity of something like `setSnappingEnabled()`. In similarly awkward situations we have things like `ViewportGadget::setDragTracking()` and `ViewportGadget::setPlanarMovement()`. I think that implies this could maybe be `setSnapping()`? Happy to hear your opinion, but I would like to aim for consistency and guessability where we can.

Reply:
Removed in rebase/rework